Source/VideoView.swift

```diff
@@ -81,6 +83,17 @@ class VideoView: UIView {
     }
 
     override func observeValue(forKeyPath keyPath: String?, of object: Any?, change: [NSKeyValueChangeKey: Any]?, context: UnsafeMutableRawPointer?) {
+        
+        if keyPath == VideoView.audioSessionVolumeKeyPath {
+            do {
+                try AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback, with: [])
+            }
+            catch let error {
+                print("Failed to start playback sound: \(error.localizedDescription)")
```

Comment:
Yes, but I was thinking about this as an "non-fatal" error, so I don't want it to fail the video playing. Or should it?